L1Trigger/TrackFindingTracklet/src/TrackletProcessor.cc

```diff
@@ -205,743 +171,419 @@ void TrackletProcessor::addInput(MemoryBase* memory, string input) {
-      edm::LogVerbatim("Tracklet") << "Will break on number of tracklets in " << getName();
-      break;
+    if (settings_.writeMonitorData("TP")) {
+      globals_->ofstream("trackletprocessor.txt") << getName() << " " << ninnerstubs   //# inner stubs
+                                                  << " " << outervmstubs_->nVMStubs()  //# outer stubs
+                                                  << " " << countteall                 //# pairs tried in TE
+                                                  << " " << stubpairs                  //# stubs pairs
+                                                  << " " << countsel                   //# tracklets found
+                                                  << endl;
     }
 
-    if (countall >= settings_.maxStep("TP")) {
-      if (settings_.debugTracklet())
-        edm::LogVerbatim("Tracklet") << "Will break on MAXTC 1";
+    if (teunitsdone && tedatabuffersdone) {
       break;
     }
-    if (settings_.debugTracklet()) {
-      edm::LogVerbatim("Tracklet") << "TrackletProcessor execute done";
-    }
-  }
-  if (countall >= settings_.maxStep("TP")) {
-    if (settings_.debugTracklet())
-      edm::LogVerbatim("Tracklet") << "Will break on MAXTC 2";
-    //break;
-  }
-
-  if (settings_.writeMonitorData("TP")) {
-    globals_->ofstream("trackletcalculator.txt") << getName() << " " << countall << " " << countsel << endl;
   }
 }
 
-void TrackletProcessor::setVMPhiBin() {
-  if (innervmstubs_.size() != outervmstubs_.size())
-    return;
-
-  for (unsigned int ivmmem = 0; ivmmem < innervmstubs_.size(); ivmmem++) {
-    unsigned int innerphibin = innervmstubs_[ivmmem]->phibin();
-    unsigned int outerphibin = outervmstubs_[ivmmem]->phibin();
-
-    unsigned phiindex = 32 * innerphibin + outerphibin;
-
-    if (phitable_.find(phiindex) != phitable_.end())
-      continue;
-
-    innervmstubs_[ivmmem]->setother(outervmstubs_[ivmmem]);
-    outervmstubs_[ivmmem]->setother(innervmstubs_[ivmmem]);
-
-    if ((layer_ == 1 && disk_ == 0) || (layer_ == 2 && disk_ == 0) || (layer_ == 3 && disk_ == 0) ||
-        (layer_ == 5 && disk_ == 0)) {
-      innerphibits_ = settings_.nfinephi(0, iSeed_);
-      outerphibits_ = settings_.nfinephi(1, iSeed_);
-
-      int innerphibins = (1 << innerphibits_);
-      int outerphibins = (1 << outerphibits_);
-
-      double innerphimin, innerphimax;
-      innervmstubs_[ivmmem]->getPhiRange(innerphimin, innerphimax, iSeed_, 0);
-      double rinner = settings_.rmean(layer_ - 1);
-
-      double outerphimin, outerphimax;
-      outervmstubs_[ivmmem]->getPhiRange(outerphimin, outerphimax, iSeed_, 1);
-      double router = settings_.rmean(layer_);
-
-      double phiinner[2];
-      double phiouter[2];
+void TrackletProcessor::writeTETable() {
+  ofstream outstubptinnercut;
+  outstubptinnercut.open(getName() + "_stubptinnercut.tab");
+  outstubptinnercut << "{" << endl;
+  //for(unsigned int i=0;i<pttableinner_.size();i++){
```

Comment:
delete commented-out code

Reply:
these are things that are not fully implemented, as it's a new part of the code which we will switch to in a future PR. I have left these lines with a comment about this.